homeassistant/components/light/qwikswitch.py

```diff
@@ -6,19 +6,16 @@
 """
 import logging
 
-import homeassistant.components.qwikswitch as qwikswitch
-
-_LOGGER = logging.getLogger(__name__)
-
 DEPENDENCIES = ['qwikswitch']
 
 
 # pylint: disable=unused-argument
 def setup_platform(hass, config, add_devices, discovery_info=None):
-    """Set up the lights from the main Qwikswitch component."""
+    """Add lights from the main Qwikswitch component."""
     if discovery_info is None:
-        _LOGGER.error("Configure Qwikswitch component failed")
+        logging.getLogger(__name__).error(
```

Comment:
Do we really need to log this? It will only happen if a user by mistake keeps a platform section for `qwikswitch` in the config under the `light` component.